### DISCUSSION TITLE
Como explicar a profissão de front-end pra mãe?

### DISCUSSION THREAD

**Discussion by luisfmsouza** [frontendbr/forum#49] [Discussão]
Completamente off-topic, mas relevante pra quem quer manter a coroa(ou qq pessoa que perguntar) atualizada sobre o que fazemos.

Como explicar a profissão de front-end pra algum parente, amigo ou conhecido, que não seja da área de tecnologia?

Por exemplo:
- Pra minha **mãe**, eu sou: **o filho que trabalha com site**.
- Pra minha **avó**, que mora em um sítio no interior, eu sou: **o netinho do computador**.
- Pro meu **tio** fazendeiro, eu sou: **o sobrinho que mexe com "esses negócios" de internet**.

E ai?

**Comment by bernardodiasc** [comment]
hehehe interessante isso, ate mesmo pq estamos acostumados com uma nomenclatura de outro idioma que nao faz nenhum sentido pra maioria das pessoas. geralmente eu aponto pro computador com facebook ou qualquer outro site aberto e falo "eu faço isso ai que vc ta vendo, que vc usa todos os dias, ajudo a construir a internet, sabe como o facebook eh feito? eu sei... etc etc etc"... realmente as vezes eh dificil explicar.

**Comment by vitorbritto** [comment]
Não vejo como @luisfmsouza. O motivo é simples: conhecimento.

A senhora sua mãe, seu avô e tio não conhecem o termo, nem as características que compõem este profissional. Está é uma área não tão popular para muitas pessoas. É diferente de "ser Engenheiro", "ser Médico" ou "ser Advogado". Até mesmo quem está na área se confunde e não sabe o que cada "rótulo" possui como habilidade. Não é verdade? =]

Meu pai, por exemplo, brinca até hoje comigo pois "gosto de ficar na frente do computador escrevendo um monte de coisas coloridas". Mal sabe ele que são Algoritmos em um editor de texto. =P

Ele é médico e tem mais de 60 anos. Imagina querer "empurrar" a definição de Front-End para ele? Não tem como. Seja por não ter o conhecimento ou por não querer este conhecimento. As pessoas mais velhas estão propensas a negar a tecnologia. Algumas mal conseguem ligar um computador. 

É mais fácil você ser amplo no assunto, falar que trabalha com Tecnologia, Internet ou usar termos populares: "sou programador", "trabalho com criação de sites", "faço sistemas para empresas".

**Comment by dionarodrigues** [comment]
Vdd @vitorbritto . Dependendo do contexto, ora utilizo terminologias mais amplas ora mais específicas. 

---

A minha mãe, por exemplo, que adora ficar na internet (interagindo no facebook, criando vídeos para o youtube...), pode não saber a fundo o que eu faço, mas ela sabe que trabalho com design (que desenho sites) e que sou front-end (que mexo com códigos pra transformar os desenhos em códigos). Hehe // O processo dela fazer essas assimilações foi algo natural, então, não me lembro sobre a dificuldade dela.

No entanto, quando um senhor taxista daqui de São Paulo me pergunta o que eu faço, por exemplo, digo que "trabalho com sites". Assim, a conversa para por aqui ou em expressões como "Que chique" ou "Que legal" ou "Que moderno". 

Quando vou para o interior, onde alguém, que parece não ter muito conhecimento em internet, me pergunta o que eu faço, digo apenas que "trabalho com computadores". Hehehee

Já para a galera mais jovem, geralmente digo que "desenvolvo sites". E por aí vai. =D

---

Mas quando digo um termo muito genérico e o ouvinte expressa alguma curiosidade a respeito, faço questão de explicar o que é. Quem sabe assim ela acaba influenciando (inspirando) outras pessoas a entrarem para a área também! =D

**Comment by ricardogouveia3** [comment]
Essa thread merece render mais.
Mas por hora, fiz [um texto sobre](https://goo.gl/Qg9sPq) e quero agradecer por serem uma das minhas referências.
